Src/MoneyFox.Windows.Tasks/LiveTilesTaskService.cs

```diff
@@ -0,0 +1,1253 @@
+													HintStyle = AdaptiveTextStyle.CaptionSubtle
+												},
+											   new AdaptiveText()
+												{
+													Text = string.Format(GetResourceString("LiveTileAccountCurrentMonthsExpenses"), await  GetMonthExpensesAsync(DateTime.Now.Month,accountid)),
+													HintStyle = AdaptiveTextStyle.CaptionSubtle
+												}
+											}
+										}
+									}
+								}
+							}
+								}
+							},
+							TileLarge = new TileBinding()
+							{
+								Content = new TileBindingContentAdaptive()
+								{
+									Children =
+							{
+								new AdaptiveGroup()
+								{
+									Children =
+									{
+										new AdaptiveSubgroup()
+										{
+											Children =
+											{
+												new AdaptiveText()
+												{
+													Text = string.Format(GetResourceString("LiveTileAccountName"),accountid.ToString()),
+													HintStyle = AdaptiveTextStyle.Caption
+												},
+											new AdaptiveText()
+												{
+													Text = string.Format(GetResourceString("LiveTileAccountBalance"),await GetLatestBalanceAsync(accountid)),
+													HintStyle = AdaptiveTextStyle.CaptionSubtle
+												},
+												new AdaptiveText()
+												{
+													Text = string.Format(GetResourceString("LiveTileAccountLastMonthsExpenses"),await GetMonthExpensesAsync(DateTime.Now.AddMonths(-1).Month,accountid)),
+													HintStyle = AdaptiveTextStyle.CaptionSubtle
+												},
+											   new AdaptiveText()
+												{
+													Text = string.Format(GetResourceString("LiveTileAccountCurrentMonthsExpenses"), await  GetMonthExpensesAsync(DateTime.Now.Month,accountid)),
+													HintStyle = AdaptiveTextStyle.CaptionSubtle
+												}
+
+											}
+										}
+									}
+								}
+							}
+								}
+							}
+						}
+					};
+
+					TileNotification tn = new TileNotification(content.GetXml());
+					TileUpdateManager.CreateTileUpdaterForSecondaryTile(accountid.ToString()).Update(tn);
+				}
+			}
+
+		}
+		private async Task<double> GetMonthExpensesAsync(int month, int accountid)
+		{
+			var currentmonth = DateTime.Now.Month;
+			double balance = 0.00;
+			IEnumerable<Payment> payments = await paymentService.GetPaymentsByAccountId(accountid);
+			foreach (Payment item in payments)
+			{
+				if (item.Data.IsRecurring)
+				{
+						GetrecurrancepaymentsMonthly(item, item.Data.RecurringPayment.Recurrence,month);
+				
+				
+				}
+				else
+				{
+					if (item.Data.Date.Month==month && item.Data.Date.Year==DateTime.Now.Year)
+					{
+						allpayment.Add(item.Data.Date, item);
+					}
+				
+				}
+			}
+			foreach (KeyValuePair<DateTime,Payment> item in allpayment)
+			{
+				balance += item.Value.Data.Amount;
+			}
+			return balance;
+		}
+		private async Task<List<string>> Get9PreviouspaymentsAsync()
```

Comment:
Nitpicking: I would write "Nine" instead of 9.

Reply:
i agree will change,

Reply:
removed the number and name nine